src/components/ChooserModal.vue

```diff
@@ -84,7 +84,7 @@
           <div v-html="$t('help.six-cc-licenses.text')" />
           <img
             alt="license openness scale"
-            src="../assets/license-openness-scale.png"
+            src="@creativecommons/cc-assets/icons/cc-license-openness-scale.png"
```

Comment:
Let's link directly to a version of this image on Wikimedia Commons, 

https://commons.m.wikimedia.org/wiki/File:Ordering_of_Creative_Commons_licenses_from_most_to_least_permissive.png

Reply:
I used the image link from this itself. can you check?